packages/babel-cli/src/babel/options.js

```diff
@@ -161,6 +161,11 @@ commander.option(
   "Delete the out directory before compilation.",
 );
 
+commander.option(
+  "--include-ignore",
+  "Include ignored files when compiling and copying non-compilable files.",
```

Comment:
🤔I don't think ignored file will be compiled when `--include-ignore` is true, it is different than `--include-dotfiles`.

Personally I prefer `--include-ignored` than `--include-ignore` because `ignore` is a verb.

Reply:
Yeah, we can drop the compiling bit.

Reply:
@JLHwung , I'll make the the change for the flag. Is the below text ok for you? Just want to confirm before pushing the change.

Include ignored files and copying non-compilable files.

Reply:
I feel like it is

> Include ignored files when copying non-compilable files.

Reply:
@JLHwung Done ✅